website/docs/r/db_instance.html.markdown

```diff
@@ -117,8 +117,7 @@ information on the [AWS
 Documentation](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_Monitoring.html)
 what IAM permissions are needed to allow Enhanced Monitoring for RDS Instances.
 * `multi_az` - (Optional) Specifies if the RDS instance is multi-AZ
-* `name` - (Optional) The DB name to create. If omitted, no database is created
-initially.
+* `name` - (Optional) Varies depending on the DB Engine. E.g.: For MySQL, this indicates that a database called 'name' should also be created with the instance. For Oracle, this is the SID of the instance. See the [AWS documentation](http://docs.aws.amazon.com/cli/latest/reference/rds/create-db-instance.html) for more details.
```

Comment:
What do you think of such documentation?

> The name of the database to create when the DB instance is created. If this parameter is not specified, no database is created in the DB instance. Note that it may vary depending on the DB Engine used. See the [AWS documentation](http://docs.aws.amazon.com/cli/latest/reference/rds/create-db-instance.html) for more details.

Reply:
Thanks for the quick response. Of course I'm biased and still prefer my version ;-) 

How about this compromise:
> The name of the database to create when the DB instance is created. If this parameter is not specified, no database is created in the DB instance. Note that this is not true for the Oracle or SQL Server engines. See the [AWS documentation](http://docs.aws.amazon.com/cli/latest/reference/rds/create-db-instance.html)  for more details.

Reply:
I would just update `Note that this is not true for the Oracle or SQL Server engines` to `Note that this does not apply for Oracle or SQL Server engines`. Thoughts? :)

Reply:
Sounds good. Thx!

Reply:
Sure, feel free to go :)

Reply:
Hi @Ninir  - Thanks for the back-and-forth on this. I'm not sure what you mean by 'free to go'?! Should I close this pull request and re-submit a new one... or something else? Thanks!

Reply:
I meant you can make the change whenever you want :)
Otherwise, I can do it: just don't want to take credit for your ideas  😄

Reply:
gotchya, thx! Done.